app/controllers/people/person/positions/minister/edit.js

```diff
@@ -0,0 +1,51 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { dropTask } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
+import { combineFullAddress } from 'frontend-contact-hub/models/address';
+
+const FINANCING_CODE = {
+  SELF_FINANCED: '997073905f839ac6bafe92b76050ab0b',
+  FOD_FINANCED: '9d6f49b3d923b437ec3a91e8b5fa6885',
+};
+
+export default class PeoplePersonPositionsMinisterEditController extends Controller {
+  @service router;
+
+  @tracked willReceiveFinancing;
+
+  @dropTask
+  *save(event) {
+    event.preventDefault();
+
+    let contacts = yield this.model.minister.contacts;
+    let address = yield contacts.firstObject.contactAddress;
+
+    if (address.isNew || address.hasDirtyAttributes) {
```

Comment:
We're editing an existing minister, so the isNew check will never be true so I would remove that.